### DISCUSSION TITLE
slowfast_r50_256p_4x16x1_256e_kinetics400_rgb || json || loss, cls_loss values

### DISCUSSION THREAD

**Discussion by sakerlrf** [open-mmlab/mmaction2#1182] [Q&A] [answered]
Hello,

Could you please explain why the loss values are that high in the given json file?

https://download.openmmlab.com/mmaction/recognition/slowfast/slowfast_r50_256p_4x16x1_256e_kinetics400_rgb/20200731_151706.log.json
In the last epoch, the loss and cls_loss was 1.39805.

Thank you!

**Comment by dreamerlin** [comment, marked as answer]
It is normal and better to focus on the acc metric

**Reply by sakerlrf** [reply]
Good to have your input. This is a controversial topic.
